app/shared/types/templateSchema.ts

```diff
@@ -75,14 +76,20 @@ ajv.addKeyword('uniquePropertyFields', {
 });
 
 ajv.addKeyword('requireContentForSelectFields', {
+  async: true,
   errors: false,
   type: 'object',
-  validate(schema: any, data: PropertySchema) {
+  async validate(schema: any, data: PropertySchema) {
     if (!schema) {
       return true;
     }
     if (['multiselect', 'select'].includes(data.type)) {
-      return !!(data.content && data.content.length);
+      if (!data.content || !data.content.length) {
+        return false;
+      }
+
+      const found = await thesauri.getById(data.content);
+      return !!found;
```

Comment:
Given the description, in the issue, this validation should throw a specific message "the thesaurus no longer exists", is it possible to change the definition of the keyword to return a ValidationError with a custom message?

Reply:
This error is quite rare, I think adding a different error message for this scenario only adds complexity and no real benefit, but thats just my opinion, so I will comment this today in the standup and see if we get an agreement

Reply:
For the record, we agreed to change the keyword to requireOrInvalidContentForSelectFields, having "validation failed: .properties[0] should pass "requireOrInvalidContentForSelectFields" keyword validation" as message